lib/screens/wallet_detail/wallet_detail_screen.dart

```diff
@@ -379,73 +301,569 @@ class _WalletDetailScreenState extends State<WalletDetailScreen> {
+  //       duration: const Duration(milliseconds: 300),
+  //       curve: Curves.easeIn,
+  //     );
+  //   }
+  //   setState(() {
+  //     _selectedFilter = orderEnum;
+  //     //_selectedUtxoList.clear();
+  //   });
+  //   await Future.delayed(const Duration(milliseconds: 100));
+  //   if (mounted) {
+  //     setState(() {
+  //       // model.UTXO.sortUTXO(_utxoList, orderEnum);
+  //     });
+  //   }
+  // }
+
+  // ViewModel
+  // _checkTxCount(int? txCount, bool isLatestTxBlockHeightZero) {
+  //   Logger.log('--> prevTxCount: $_prevTxCount, wallet.txCount: $txCount');
+  //   Logger.log(
+  //       '--> prevIsZero: $_prevIsLatestTxBlockHeightZero, wallet.isZero: $isLatestTxBlockHeightZero');
+  //
+  //   /// _walletListItem의 txCount, isLatestTxBlockHeightZero가 변경되었을 때만 트랜잭션 목록 업데이트
+  //   if (_prevTxCount != txCount ||
+  //       _prevIsLatestTxBlockHeightZero != isLatestTxBlockHeightZero) {
+  //     List<TransferDTO>? newTxList = _model.getTxList(widget.id);
+  //     if (newTxList != null) {
+  //       print('--> [detail화면] newTxList.length: ${newTxList.length}');
+  //       _txList = newTxList;
+  //       setState(() {});
+  //     }
+  //   }
+  //   _prevTxCount = txCount;
+  //   _prevIsLatestTxBlockHeightZero = isLatestTxBlockHeightZero;
+  // }
+  // // setListener end
+
+  // VieModel
+  // void _removeFaucetTooltip() {
+  //   // if (_overlayEntry != null) {
+  //   //   _faucetTipVisible = false;
+  //   //   _overlayEntry!.remove();
+  //   //   _overlayEntry = null;
+  //   // }
+  //   setState(() {
+  //     _faucetTipVisible = false;
+  //     _faucetTooltipVisible = false;
+  //   });
+  // }
+
+  // ViewModel
+  // getUtxoListWithHoldingAddress(
+  //     List<UTXO> utxoEntities,
+  //     WalletListItemBase walletBaseItem,
+  //     String accountIndexField,
+  //     String changeField,
+  //     WalletType walletType) {
+  //   List<model.UTXO> utxos = [];
+  //   for (var utxo in utxoEntities) {
+  //     String ownedAddress = walletBaseItem.walletBase.getAddress(
+  //         DerivationPathUtil.getAccountIndex(walletType, utxo.derivationPath),
+  //         isChange: DerivationPathUtil.getChangeElement(
+  //                 walletType, utxo.derivationPath) ==
+  //             1);
+  //
+  //     final tags = _model.loadUtxoTagListByTxHashIndex(widget.id, utxo.utxoId);
+  //
+  //     utxos.add(model.UTXO(
+  //       utxo.timestamp.toString(),
+  //       utxo.blockHeight.toString(),
+  //       utxo.amount,
+  //       ownedAddress,
+  //       utxo.derivationPath,
+  //       utxo.transactionHash,
+  //       utxo.index,
+  //       tags: tags,
+  //     ));
+  //   }
+  //
+  //   setState(() {
+  //     _utxoList = utxos;
+  //     _isUtxoListLoadComplete = true;
+  //   });
+  // }
+
+  @override
+  Widget build(BuildContext context) {
+    return ChangeNotifierProxyProvider<AppStateModel, WalletDetailViewModel>(
+      create: (_) => WalletDetailViewModel(widget.id),
```

Comment:
WalletDetailViewModel 생성할 때 AppStateModel을 넘기지 않는 이유가 있을까요?
```suggestion
      create: (_) => WalletDetailViewModel(widget.id),
```

Reply:
ChangeNotifierProxyProvider<AppStateModel, WalletDetailViewModel>
- 위와 같이 생성시 AppStateModel의 변화를 감지하고 update: (_, appStateModel, viewModel)를 통해 전달받습니다.
- update: (_, appStateModel, viewModel)는 처음 뷰모델 생성시에 호출되고 AppStateModel 내에서 notifyListener() 실행이 감지될 때마다 호출됩니다.